easybuild/easyconfigs/m/mutil/mutil-1.822.3-intel-2016a.eb

```diff
@@ -0,0 +1,34 @@
+name = 'mutil'
+version = '1.822.3'
+
+homepage = 'http://people.nas.nasa.gov/~kolano/projects/mutil.html'
+description = """Mutil is a set of standard utilities that have been parallelized to maximize performance on 
+modern file systems. These currently include multi-threaded drop-in replacements for cp and md5sum from
+GNU coreutils, which have achieved 10/30x rates on one/many nodes.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+sources = [
+    "coreutils-8.22.tar.xz",  # must be first
+    SOURCE_TGZ,
+]
+source_urls = [
+    ('http://sourceforge.net/projects/mutil/files', 'download'),
+    'http://ftpmirror.gnu.org/coreutils',
+]
+
+dependencies = [
+    ('gnutls', '3.3.21'),
+    ('libgpg-error', '1.21'),
+    ('libgcrypt', '1.6.5'),
+]
+
+builddependencies = [('Automake', '1.14')]
```

Comment:
It's a patch specific for Automake 1.14 and it doesn't work for 1.15. Only Automake is needed.